src/utils/validationUtil.ts

```diff
@@ -51,7 +60,7 @@ export class ValidationUtil {
     //   this.ivaFailedHasRequiredFields(payload.testTypes);
     // }
 
-    this.validateCentralDocs(payload.testTypes);
+    this.validateCentralDocs(payload.testTypes ?? []);
```

Comment:
same here, this one might be more just a type issue tho, has that smell.

Reply:
As far as I can tell testTypes is required so I agree, I don't think the ?? [ ] is required.